trade-finance-manager-api/api-tests/v1/deal-cancellation/deal-cancellation.api-test.ts

```diff
@@ -76,36 +85,44 @@ describe('/v1/deals/:id/cancellation', () => {
         jest.resetAllMocks();
       });
 
+      withTeamAuthorisationTests({
```

Comment:
Do we have other team specific routes? I believe there are some for underwriters at least.

If so, seems `withTeamAuthorisationTests` is a new pattern. If we want to use this new pattern we should either update the other tests or use the original pattern, rather than introducing a different way of testing the same thing.

Ignore me if i'm wrong or missing something 😊